docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -33,8 +33,8 @@ The architecture has the following components:
   * **Front-end configuration**. Associates the public IP address with the load balancer.
   * **Back-end address pool**. Contains the network interfaces (NICs) for the VMs that will receive the incoming traffic.
 * **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
-* **VM Scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out anually, or based on predefined rules.
-* **Availability set**. The [availability set][availability set] contains the VMs, making the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. In order for the SLA to apply, the availability set must include a minimum of two VMs. Availability sets are implicit in scale sets. If you create VMs outside a scale set, you need to create the availability set independently.
+* **VM scale set**. A [VM scale set][vm-scaleset] is a set of identical VMs used to host a workload. Scale sets allow the number of VMs to be scaled in or out annually, or based on predefined rules.
```

Comment:
annually -> manually

Unless it's a Black Friday thing 😸

Reply:
Oops - misread that typo :-)